server/src/test/proto/spine/test/event/events.proto

```diff
@@ -97,3 +99,105 @@ message EBTaskAdded {
 message ProjectFinished {
     ProjectId project_id = 1;
 }
+
```

Comment:
Let's extract into a separate file. And let's have a file-level comment on the purpose of those events.